draft-ietf-quic-recovery.md

```diff
@@ -1119,7 +1125,7 @@ PeerNotAwaitingAddressValidation():
          has received 1-RTT ACK
 
 SetLossDetectionTimer():
-  loss_time, _ = GetEarliestLossTime()
+  loss_time, _ = GetEarliestTimeAndSpace(loss_times)
```

Comment:
`loss_time` is a global variable, `loss_times` is not defined.
This should be 
```
earliest_loss_time, _ = GetEarliestTimeAndSpace(loss_time)
```

Reply:
Good catch, fixed.

Reply:
Did you forget to push your fix?

Reply:
I fixed it by defining loss_times, but I'll revert that if you don't like it.  I prefer plurals, but I'll keep them consistent and singular for now.